layer.yaml

```diff
@@ -15,6 +15,11 @@ options:
     puppet-version: '3'
     silent: True
 defines:
+  install_java:
+    type: string
+    default: 'openjdk-8-jdk'
```

Comment:
On another subject, the default value of "openjdk-8-jdk" will break for trusty, because there is no openjdk-8-jdk deb package. There is however the meta-package "default-jdk" that will give you openjdk-8 and jre on xenial and openjdk-7 and jre on trusty.

If we need java 8 for bigtop, we should take proper action to update our trusty charms.